src/datacatalogtordf/document.py

```diff
@@ -0,0 +1,141 @@
+"""Document module for mapping a document to rdf.
+
+This module contains methods for mapping a document object to rdf
+according to the
+`dcat-ap-no v.2 standard <https://doc.difi.no/review/dcat-ap-no/#klasse-distribusjon>`__
+
+Example:
+    >>> from datacatalogtordf import Document
+    >>> # Create a document:
+    >>> document = Document()
+    >>> document.identifier = "http://example.com/documents/1"
+    >>> document.title = {"en": "The Python Language Reference Manual"}
+"""
+from __future__ import annotations
+
+from typing import Optional
+
+from rdflib import BNode, DCTERMS, FOAF, Graph, Literal, Namespace, RDF, URIRef
+
+from datacatalogtordf.uri import URI
+
+DCAT = Namespace("http://www.w3.org/ns/dcat#")
+
+
+class Document:
+    """A class representing a foaf:Document.
+
+    Attributes:
+        identifier (URI): A URI uniquely identifying the document
+        title (dict): A title given to the document. key is langauge code
+        language (str): The organzation's identifier
+        format (str): a link to a concept designating the type of the document
+    """
+
+    slots = ("_identifier", "_title", "_language", "_format")
+
+    _identifier: URI
+    _title: dict
+    _language: str
+    _format: str
```

Comment:
Iflg [dcat-ap-no](https://data.norge.no/specification/dcat-ap-no/#Dokument) har foaf:Document følgande properties:
- språk,
- tittel

Iflg [modelldcat-ap-no](https://informasjonsforvaltning.github.io/modelldcat-ap-no/#Dokument-format) har foaf:Document følgande properties _i tillegg_:
- format 

Derfor er eg litt usikker på om `format` høyrer heime i dette biblioteket...

Reply:
Burde vi kanskje ha et eget sted å foaf-ting? Kanskje et nytt lite bibliotek? Eller hvordan løser vi dette?

Reply:
Enig i at dette egentlig ikke hører hjemme i dcat-ap-no. Det er jo eksterne klasser..

Reply:
Men skal vi sette opp en task på å lage et nytt bibliotek og trekke ut disse klassene? Det kan vi kanskje gjøre...

Reply:
foaf:Document høyrer heime i datacatalogtordf, men datacatalogtordf bør kanskje ikkje ha properties på foaf:Document som ikkje står i dcat-ap-no. Mitt forslag vil derfor vere å fjerne format.

Reply:
Hmm.. :thinking:  Men format trengs av modelldcatapno.. Hvordan gjør vi det da? :thinking:

Reply:
En løsning kunne jo kanskje vært at vi satte opp et eksternt bibliotek med foaf slik som vi har med skos:Concept. Og så la en avhengighet til hvert av bibliotekene der. Selvfølgelig under forutsetning av at du synes dette er greit.. :slightly_smiling_face:

Reply:
Min første tanke er å spesialisere Document i modelldcatapno og legge på ekstra property der.

Reply:
Ah, akkurat , det kan jeg gjøre! :+1:

Reply:
Så kan vi eventuelt senere ta denne saken med eget bibliotek for foaf. :slightly_smiling_face:

Reply:
Kanskje. Men du finner ikkje dct:format som ein av egenskapane til til foaf:Document. Men alt er jo lov, så det er ikkje feil, berre rart. Ref http://xmlns.com/foaf/spec/#term_Document

Reply:
Da har jeg fjernet dct:format @stigbd ! :slightly_smiling_face: